ogc/src/main/scala/geotrellis/server/ogc/OgcSource.scala

```diff
@@ -55,7 +55,7 @@ trait OgcSource {
   def attributes: Map[String, String]
   def resampleMethod: ResampleMethod
   def overviewStrategy: OverviewStrategy
-  def timeInterval: Option[OgcTimeInterval]
+  def timeInterval: OgcTime
```

Comment:
If we do end up keeping `OgcTime`, then this def should probably be renamed as its no longer necessarily an interval. Perhaps `timeInterval` -> `time`?